.yarnrc.yml

```diff
@@ -1,2 +1,4 @@
 yarnPath: .yarn/releases/yarn-4.3.1.cjs
 nodeLinker: node-modules
+enableHardenedMode: false
+enableImmutableInstalls: false
```

Comment:
These two added as the yarn restrictions for public repos set by default after our npm -> yarn swap (https://github.com/codecov/gazebo/pull/3016) were not allowing us to add new packages.